### DISCUSSION TITLE
Is it possible to run Directus on a local computer offline?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#9230] [Archive]
Directus works fine on my computer with local code and local database when connected to Internet.

Offline, `npx directus start` fetches packages (as usual) on my computer for minutes but then stops because it cannot reach some servers for packages updates making it necessary to be online. I do not need theses updates every time I want to run Directus.

As everything is on my computer (data + code), what is the command to run directus when completely offline?

**Comment by nickrum** [comment]
After installing Directus with `npm init directus-project example-project`, the start command shouldn't download/install anything. Running `npx directus start` without a network connection works fine on my end.

**Comment by Pachat** [comment]
Thanks for your reply. I did a proper reinstall and it works perfectly.

The reason I had a problem is that I downloaded the whole server
installation directly and therefore had files not in concordance with my
local node version which is different from the one on the server.
